lambda-views/models/page_views.sql

```diff
@@ -0,0 +1,32 @@
+{{
+    config(
+        materialized='view'
+    )
+}}
+
+with current_view as (
+
+    select * from {{ ref('page_views_current') }}
+    where max_collector_tstamp >= dateadd('hour', -2, current_timestamp())
```

Comment:
Should we be using `run_started_at` ([docs](https://docs.getdbt.com/reference/dbt-jinja-functions/run_started_at/))
```suggestion
    where max_collector_tstamp >= dateadd('hour', -2, '{{ run_started_at }}')
```

Let's say we run this at 12pm. Something goes wrong, and we don't have any more dbt runs.

If we use `current_timestamp()` here, at 3pm:
- The view will only select data from 1pm onwards
- The table will select data from before 1pm, BUT, since we last ran at midday, there's no data in the table between 12pm and 1pm.


By harcoding the cutoff time as `12pm`, instead at 3pm:
- The view will continue to select data from 12pm onwards (this just means the view does more work)
- The table will continue to select data from before 12pm